modules/installation-ibm-z-user-infra-machines-iso.adoc

```diff
@@ -115,16 +115,25 @@ Write all options in the parameter file as a single line and make sure you have
 When you install with multiple paths, you must enable multipathing directly after the installation, not at a later point in time, as this can cause problems.
 ====
 ... Set the install device as: `coreos.inst.install_dev=/dev/disk/by-id/scsi-<serial_number>`.
+
+. Optional: Create a generic.ins file:
 +
-[NOTE]
-====
-If additional LUNs are configured with NPIV, FCP requires `zfcp.allow_lun_scan=0`. If you must enable `zfcp.allow_lun_scan=1` because you use a CSI driver, for example, you must configure your NPIV so that each node cannot access the boot partition of another node.
-====
-... Leave all other parameters unchanged.
+Some installation methods also require a generic.ins file with a mapping of the location of the installation data in the file system of the Hardware Management Console (HMC), the DVD, or the FTP server and the memory locations where the data is to be copied. A sample generic.ins file is provided with the {op-system-base} installation media. The file contains file names for the initial RAM disk (initrd.img), kernel image (kernel.img), and parameter (generic.prm) files and a memory location for each file.
```

Comment:
```suggestion
Some installation methods also require a `generic.ins` file with a mapping of the location of the installation data in the file system of the Hardware Management Console (HMC), the DVD, or the FTP server and the memory locations where the data is to be copied. A `sample generic.ins` file is provided with the {op-system-base} installation media. The file contains file names for the initial RAM disk (`initrd.img`), the kernel image (`kernel.img`), and the parameter , (`generic.prm`) files and a memory location for each file.
```

Reply:
I couldn't find clear guidance on this but I think in general the appositives inside of ( ) I _think_ is to be avoided. https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses#:~:text=Avoid%20using%20parentheses%20in%20general%20text%20because,accurate%20if%20you%20omit%20the%20parenthetical%20text. 

I couldn't come up with a great solution other than `, ;` to separate the list.

Reply:
@philc7777 @alkleinrh Do we really require the names in parentheses or can I simply remove them?